lib/node_modules/@stdlib/stats/base/nanvariancech/test/test.nanvariancech.js

```diff
@@ -313,9 +485,8 @@ tape( 'the function supports view offsets', function test( t ) {
 	]);
 
 	x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
-	N = floor(x1.length / 2);
 
-	v = nanvariancech( N, 1, x1, 2 );
+	v = nanvariancech( 4, 1, x1, 2 );
```

Comment:
```suggestion
	v = nanvariancech( 5, 1, x1, 2 );
```
kindly follow the commented indices, same comment for all other instances